www/src/scss/navigator/common/_verse-block.scss

```diff
@@ -88,12 +88,17 @@
     &.shabad-li {
       padding: 5px 10px;
     }
-
     div {
       display: inline-flex;
       a {
         padding: 5px;
         font-size: 1rem;
+
+        .eng-verse {
+          color: $rust;
+          display:block;
```

Comment:
indentation issue

Reply:
Looks fine to me veerji, it has two spaces everywhere.